dev/DropDownButton/DropDownButton.xaml

```diff
@@ -62,6 +62,9 @@
                                         <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ContentPresenter" Storyboard.TargetProperty="Foreground">
                                             <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource ButtonForegroundPressed}" />
                                         </ObjectAnimationUsingKeyFrames>
+                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ChevronTextBlock" Storyboard.TargetProperty="Foreground">
+                                            <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource SplitButtonForegroundSecondaryPressed}" />
```

Comment:
Can we make a dropdown-specific brush for this one? DropDownButton shouldn't really reference a SplitButton* brush.

Reply:
Does that mean we should create brushes for other elements as well? Since we are reference Button* brushes.

Reply:
Spoke offline with @ranjeshj , it makes sense to create a `DropDownButton_themeresources.xaml`, and re-naming all the brushes to `DropDownButton*`

@YuliKl @teaP does that work?